apps/jackbnb-fe/src/components/Footer/index.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import Form from '../Form'
+import FormButton from '../SecondaryButton';
+import FooterStyles from './index.module.css';
```

Comment:
Recorda el espacio entre los import de las librerias y los import de otros archivos
```suggestion
import React from 'react';

import Form from '../Form'
import FormButton from '../SecondaryButton';
import FooterStyles from './index.module.css';
```